vanetza/security/encap_request.hpp

```diff
@@ -9,13 +9,29 @@ namespace vanetza
 namespace security
 {
 
+/**
+ * Security services to invoke when GN MIB's itsGnSecurity is enabled.
+ *
+ * When a SecurityProfile parameter is set to Default, the value Sign is used as a reasonable default.
+ */
+enum class SecurityProfile
+{
+    Default,
+    Sign
+    // TODO: Add Encrypt and SignAndEncrypt when encryption is supported
+};
+
 /**
  * Input data for encapsulating a packet in a secured message
  * \see TS 102 723-8 for SN-ENCAP.request
  */
 struct EncapRequest {
     DownPacket plaintext_payload; // mandatory
+    boost::optional<SecurityProfile> sec_services; // optional
```

Comment:
I know that *sec_services* is defined in TS 102 723-8 but it does not make any sense if we are not going to implement encryption. Do you have plans to implement encryption? Is anybody using this feature at all?
Since `SecurityProfile::Default` exists, I see no point in wrapping it into a boost::optional.